src/main/java/ru/vk/itmo/test/reference/ReferenceServer.java

```diff
@@ -117,8 +164,23 @@ public Response entity(Request request, @Param(value = "id") String id) {
         }
     }
 
+
+    private String getNodeByEntityId(String id) {
+        Integer maxHash = null;
+        int nodeId = 0;
+        for (int i = 0; i < config.clusterUrls().size(); i++) {
```

Comment:
Это микрооптимизация, конечно, но в кластере есть по крайней мере мы и проверку на `null` в `if` хотелось бы убрать из цикла, поэтому, может, так?
```suggestion
        int maxHash = Hash.murmur3(config.clusterUrls().get(0) + id);
        int nodeId = 0;
        for (int i = 1; i < config.clusterUrls().size(); i++) {
```